rct229/ruletest_engine/ruletest_jsons/system_types/System_11.1_VAV_SZ.json

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "ASHRAE229 1",
+    "ruleset_model_instances": [
+        {
+            "id": "RMD 1",
+            "buildings": [
+                {
+                    "id": "Building 1",
+                    "building_open_schedule": "Required Building Schedule 1",
+                    "building_segments": [
+                        {
+                            "id": "Building Segment 1",
+                            "zones": [
+                                {
+                                    "id": "Thermal Zone 1",
+                                    "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                    "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                    "terminals": [
+                                        {
+                                            "id": "VAV Air Terminal 1",
+                                            "is_supply_ducted": true,
+                                            "type": "VARIABLE_AIR_VOLUME",
+                                            "served_by_heating_ventilation_air_conditioning_system": "System Type 11.1"
+                                        }
+                                    ]
+                                }
+                            ],
+                            "heating_ventilation_air_conditioning_systems": [
+                                {
+                                    "id": "System Type 11.1",
+                                    "cooling_system": {
+                                        "id": "CHW Coil 1",
+                                        "cooling_system_type": "FLUID_LOOP",
+                                        "chilled_water_loop": "Chiller Loop 1"
+                                    },
+                                    "heating_system": {
+                                        "id": "HHW Coil 1",
+                                        "heating_system_type": "FLUID_LOOP",
```

Comment:
Could you check if this needs to be ```ELECTRIC_RESISTANCE``` according to [HERE](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_heating_type_elec_resistance.md#:~:text=heating_system_b.heating_system_type%20%3D%3D%20%22ELECTRIC_RESISTANCE%22)?

Reply:
I believe this has been updated. I hope so!